Makefile

```diff
@@ -20,4 +20,9 @@ checks:
 	gometalinter --vendor --disable-all --enable=vet --enable=gofmt --enable=golint --enable=deadcode --enable=varcheck --enable=structcheck --enable=misspell --deadline=15m ./...
 
 ci-docker:
-	docker run -it -e BUILD_NUMBER=${TRAVIS_BUILD_NUMBER} -v $(CURDIR):/go/src/github.com/lawrencegripper/azbrowse golang:1.10 bash -f /go/src/github.com/lawrencegripper/azbrowse/scripts/release.sh
\ No newline at end of file
+	docker run -it -e BUILD_NUMBER=${TRAVIS_BUILD_NUMBER} -v $(CURDIR):/go/src/github.com/lawrencegripper/azbrowse golang:1.10 bash -f /go/src/github.com/lawrencegripper/azbrowse/scripts/release.sh
+
+swagger-codegen:
+	go run cmd/swagger-codegen/main.go --output-file ./internal/pkg/handlers/swagger.generated.go 
+	gofmt -s -w internal/pkg/handlers/swagger.generated.go
```

Comment:
[nit] I think it would be good to add `go build internal/pkg/handlers/swagger.generated.go internal/pkg/handlers/types.go` here to check the output is building and isolate gen errors from other build errors. 

To make this line work I've pulled out `Register` into `Register.go` so we don't build all the other handlers too.

Reply:
Do you think we could also add a test which loads `swagger.generated.go` and checks it doesn't panic? I guess the aim would be to automate the check that one of the paths isn't parsable by `mustGetEndpointInfoFromURL`

Reply:
Happy to pick both of these up btw

Reply:
I'm happy for you to pick these up - sounds like you have a clear vision of what you want :-)

Reply:
Cool, done :)